Makefile

```diff
@@ -9,9 +9,9 @@ export IMAGE_VERSION = v1.16.0
 export SIMPLE_VERSION = $(shell (test "$(shell git describe)" = "$(shell git describe --abbrev=0)" && echo $(shell git describe)) || echo $(shell git describe --abbrev=0)+git)
 export GIT_VERSION = $(shell git describe --dirty --tags --always)
 export GIT_COMMIT = $(shell git rev-parse HEAD)
-export K8S_VERSION = 1.21
+export K8S_VERSION = 1.22
```

Comment:
It should not be 1.23?
See: https://github.com/operator-framework/operator-sdk/blob/master/go.mod#L35-L40